crates/partition-store/src/tests/invocation_status_table_test/mod.rs

```diff
@@ -212,6 +215,19 @@ async fn test_migration() {
     .unwrap();
     txn.commit().await.unwrap();
 
+    // Now run the migrations
+    rocksdb.verify_and_run_migrations().await.unwrap();
+    let partition_id = rocksdb.partition_id();
+
+    assert_eq!(
+        LastExecutedMigration::from(
+            get_last_executed_migration(&mut rocksdb, partition_id)
+                .await
+                .unwrap()
+        ),
+        LATEST_MIGRATION
+    );
+
     // Make sure we can read without mutating
```

Comment:
This and the "Now reading should perform the migration" comments are now obsolete since the migration happens upfront, right? Can you please update the remaining section of the test?